Client/metrics.yaml

```diff
@@ -1414,6 +1414,72 @@ sync:
     notification_emails:
       - fx-ios-data-stewards@mozilla.com
     expires: "2022-07-01"
+  registration_view:
+    type: event
+    description: |
+      Recorded when user opens a registration fxa page
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXIOS-3719
```

Comment:
Question for bugs in metrics.yaml - Should we put the Github [issue](https://github.com/mozilla-mobile/firefox-ios/issues/9946) since that one is public?

Reply:
I guess that would be then a @dnarcese question 😄

Reply:
If it has a github issue lets link that

Reply:
I updated our documentation here: https://github.com/mozilla-mobile/firefox-ios/wiki/Adding-Glean-Telemetry-Events
To reflect this conversation